docs/src/main/sphinx/connector/hive.rst

```diff
@@ -654,6 +654,8 @@ Property Name                                        Description
                                                      required when running in a GovCloud region.
                                                      Example: ``us-gov-east-1``
 
+``hive.metastore.glue.proxy-api-id``                 ID of Glue Proxy API.
```

Comment:
very sparse docs - I'm not sure if I'll be able to tell when to set this or not. Can you mention when one might want to set this and to what value?

Reply:
Yeah very sparse even in AWS docs. Added a little but probably only make sense to those who know AWS VPC & networking.